### DISCUSSION TITLE
I cannot use student co-pilot because my information is not verified.

### DISCUSSION THREAD

**Discussion by kutsalkiremitci** [community/community#38107] [Copilot Conversations]
Hello, even though I uploaded all the photos that I am a student for github co-pilot, github cannot process my information correctly and I cannot become a github co-pilot owner how can I do it?

NOTE: I provide the requested information,
I shared my location services via Google.
I uploaded my curriculum or School Card to the System, but it still doesn't happen.

**Comment by david-engelmann** [comment]
It does take some time for them to process the request, so If you just submitted, be patient
